packages/workers/src/block-runner.ts

```diff
@@ -71,60 +78,43 @@ export class BlockRunner {
     return true
   }
 
-  async start(startBlock: number) {
```

Comment:
When doing the logging I was getting a bit confused with all the names `startBlock`, `processingBlockNumber` and `latestBlockNumber`.

Since `startBlock` is used to initialise `processingBlockNumber`, I made it so that is done at constructor time, which has the added advantage that `processingBlockNumber` is now part of the class state and it is easier to access it for logging the value.